data/json/items/magazines.json

```diff
@@ -65,14 +352,314 @@
+        "material" : "steel",
+        "volume" : 1,
+        "weight" : 120,
+        "ammo_type" : "32",
+        "capacity" : 20,
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },
+    {
+        "type" : "MAGAZINE",
+        "id" : "ak74mag",
+        "price" : 1500,
+        "name" : "AK-74M magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard magazine made for the AK-74M rifle, made of steel-reinforced polymer.  Holds 30 rounds",
+        "material" : ["steel", "plastic"],
+        "volume" : 2,
+        "weight" : 230,
+        "to_hit" : -2,
+        "ammo_type" : "545x39",
+        "capacity" : 30,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ]
+    },
+    {
+        "type" : "MAGAZINE",
+        "id" : "rpk74mag",
+        "price" : 8500,
+        "//" : "Technically it's meant for the RPK-74, but can always rename if we add an RPK.",
+        "name" : "AK-74M extended magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A 45-round magazine compatible with the AK-74M rifle, though originally designed for the RPK-74 light machine gun.",
+        "material" : "plastic",
+        "volume" : 3,
+        "weight" : 300,
+        "ammo_type" : "545x39",
+        "capacity" : 45,
+        "reliability" : 8,
+        "reload_time" : 130,
+        "flags": [ "BULKY_MAG" ]
+    },
+    {
+        "type" : "MAGAZINE",
+        "id" : "m1918mag",
+        "//" : "Compatable magazines made for reproductions exist, and the standard-capacity mags are fairly cheap compared to variants.",
+        "price" : 3600,
+        "name" : "BAR magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with the Browning Automatic Rifle, holds 20 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 200,
+        "ammo_type" : "3006",
+        "capacity" : 20,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags": [ "COMPACT_MAG" ],
+        "alternatives" : [
+            [ "223", [ "stanag30", "stanag10", "survivor223mag" ] ]
+        ]
+    },
+    {
+        "type" : "MAGAZINE",
+        "id" : "m1918bigmag",
+        "price" : 8500,
+        "name" : "BAR extended magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An aftermarket magazine for use with the Browning Automatic Rifle, holds 30 rounds.",
+        "material" : "steel",
+        "volume" : 3,
+        "weight" : 300,
+        "ammo_type" : "3006",
+        "capacity" : 30,
+        "reliability" : 7,
+        "reload_time" : 120,
+        "flags": [ "COMPACT_MAG" ],
+        "alternatives" : [
+            [ "223", [ "stanag30", "stanag10", "survivor223mag" ] ]
+        ]
+    },
+    {
+        "type" : "MAGAZINE",
+        "id" : "belt308",
```

Comment:
Technically a reusable belt would be effectively just another magazine though, though potentially one that's a pain in the ass to reload. :V